docs/run-platform-server/configuring.mdx

```diff
@@ -79,10 +78,10 @@ If not using the `NETWORK` variable, then must provide the `STELLAR_CORE_URL` va
 
 - If you have configured your deployment to perform the ingestion role, then it is **strongly** recommended to review [Ingestion](./ingestion.mdx) first and [Filtering](./ingestion-filtering) second and factor that into configuration parameters to achieve best performance related to your application requirements before proceeding further.
 
-- `TX_SUB_ENABLED`, optional, enabled as TRUE by default. Controls whether Horizon will accept HTTP requests to `/tx` API endpoint and forward to network. Refer to [Channel Accounts](../encyclopedia/channel-accounts.mdx) for some recommendations on optional client transaction submission optimizations.
+- `DISABLE_TX_SUB`, optional, set as FALSE by default. Controls whether Horizon will accept HTTP requests to `/tx` API endpoint and forward to network. Refer to [Channel Accounts](../encyclopedia/channel-accounts.mdx) for some recommendations on optional client transaction submission optimizations.
 
-  - When enabled, requires **live** ingestion process to be running on the same database because Horizon depends on new ledgers from network to confirm a transaction submission status, Horizon will report startup error if it detects no **live** ingestion. Requires `NETWORK` or `STELLAR_CORE_URL` be defined for access to network.
-  - When disabled with FALSE, Horizon will return 405 on POSTs to /tx.
+  - When set to false, it requires **live** ingestion process to be running on the same database because Horizon depends on new ledgers from network to confirm a transaction submission status, Horizon will report startup error if it detects no **live** ingestion. Requires `NETWORK` or `STELLAR_CORE_URL` be defined for access to network.
```

Comment:
```suggestion
  - When set to FALSE, it requires **live** ingestion process to be running on the same database because Horizon depends on new ledgers from network to confirm a transaction submission status, Horizon will report startup error if it detects no **live** ingestion. Requires `NETWORK` or `STELLAR_CORE_URL` be defined for access to network.
```

Just to be consistent.